endpoints/README.MD

```diff
@@ -27,6 +27,46 @@ Note that not all notice types are supported in Hilma. Trying to create a non su
 - Social notices 20 and 21 and 33 to 25A
 
 Notice types E1 to E5 are not yet supported.
+  - Will be supported from 1.1.2024
```

Comment:
Some example of the model would be nice, and documenting that key of the statistics model is the lot id